src/components/BorrowerProfile/LoanDescription.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+	<section>
+		<p
+			class="tw-text-h2"
+		>
+			More about this loan
```

Comment:
Should this be "{{ borrowerName }}'s story"?

Reply:
You're right, I was looking at the older sigma file. Will update!